### ISSUE TITLE
Timeouts

### ISSUE THREAD

**Issue by potatoqualitee** [PowerShell/PowerShellGallery#173]
For the past couple months, I've had timeouts on both a Windows 10 workstation in the EU and on GitHub Linux runners, somewhere in the world. 

Publish takes forever for dbatools, which is fine, but sometimes it says it times out then i'll get an email saying that there's a new release. I've upped my timeout to 400 and now I'm trying for 600. Is this expected behavior for larger modules?

Thanks,
Chrissy

**Comment by potatoqualitee** [comment]
Cool, thanks. Yes, my workaround works. Looking forward to a resolution!

**Comment by iammerus** [comment]
![image](https://user-images.githubusercontent.com/22162227/127603525-37ca12eb-e723-45a6-a1f0-0294cc6510f5.png)

Been also experiencing timeouts while trying to install modules from PSGallery lately

**Comment by potatoqualitee** [comment]
yeah, this is very impactful and has been going on a surprisingly long time :/
